cypress/integration/plugins/observability-dashboards/7_integrations.spec.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+
+import { BASE_PATH } from '../../../utils/base_constants';
+
+const testInstanceName = 'test_integration_cypress';
+
+const moveToAvailableNginxIntegration = () => {
+  cy.visit(`${BASE_PATH}/app/integrations#/available/nginx`);
+};
+
+const moveToAddedIntegrations = () => {
+  cy.visit(`${BASE_PATH}/app/integrations#/installed`);
+};
+
+const createSamples = () => {
+  moveToAvailableNginxIntegration();
+  cy.get('[data-test-subj="try-it-button"]').click();
+  cy.get('.euiToastHeader__title').should('contain', 'successfully');
+};
+
+describe('Add nginx integration instance flow', () => {
+  it('Navigates to nginx page and triggers the adds the instance flow', () => {
+    createSamples();
```

Comment:
Shall we cleanup the samples after each test case?

Reply:
Added a teardown method that runs afterEach and moved createSamples to beforeEach